nifi/nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/RouteOnAttribute.java

```diff
@@ -236,7 +259,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
 
             // now transfer any clones generated
             for (final Map.Entry<Relationship, FlowFile> entry : transferMap.entrySet()) {
-                logger.info(this + " cloned " + flowFile + " into " + entry.getValue() + " and routing clone to relationship " + entry.getKey());
+                logger.info("Cloned {} into {} and routing clone to relationship {}", new Object[]{ flowFile, entry.getValue(), entry.getKey() });
```

Comment:
@madrob the ProcessorLog that is used here automatically adds it in so that it is done consistently and without the developers having to remember to log it. So if we add 'this' to the beginning, it ends up getting logged twice.